Sources/StreamChatSwiftUI/ChatChannelList/ChatChannelListViewModel.swift

```diff
@@ -433,7 +443,7 @@ open class ChatChannelListViewModel: ObservableObject, ChatChannelListController
                 return ChannelSelectionInfo(
                     channel: channel,
                     message: message,
-                    searchType: .channels
+                    searchType: .messages
```

Comment:
UI was flickering when changing the search type because incorrect search type was set here.